app/tools/jsondb.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+
+class Database:
+    def __init__(self):
+        self.files = {}
+
+    def use(self, name_space):
+        self.name_space = name_space
+
+    def create(self):
+        if self.name_space in self.files:
+            return False
```

Comment:
Co tak na prawde w tym warunku robi zmienna klasowa name_space bo w mojej ocenie za bardzo nic, a za każdym razem ją sprawdzasz. Jeśli zrobili byśmy tak jak proponuje komentarz wyżej to w ogóle  warunki sprawdzania name_space przestają mieć sens no i oczywiście mniej kodu